modules/monitoring-specifying-how-a-service-is-monitored.adoc

```diff
@@ -9,12 +9,22 @@
 [role="_abstract"]
 To use the metrics exposed by your service, you must configure {product-title} monitoring to scrape metrics from the `/metrics` endpoint. You can do this using a `ServiceMonitor` custom resource definition (CRD) that specifies how a service should be monitored, or a `PodMonitor` CRD that specifies how a pod should be monitored. The former requires a `Service` object, while the latter does not, allowing Prometheus to directly scrape metrics from the metrics endpoint exposed by a pod.
 
+[NOTE]
+====
+In {product-title}, you can use the `tlsConfig` property for a `ServiceMonitor` resource to specify the TLS configuration to use when scraping metrics from an endpoint. The `tlsConfig` property is not yet available for `PodMonitor` resources. If you need to use a TLS configuration when scraping metrics, you must use the `ServiceMonitor` resource.
```

Comment:
> The `tlsConfig` property is not yet available for `PodMonitor` resources
checked in 4.13, I see there is `tlsConfig` setting for `PodMonitor`
```
$ oc get clusterversion
NAME      VERSION                              AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.13.0-0.nightly-2023-06-07-120440   True        False         73m     Cluster version is 4.13.0-0.nightly-2023-06-07-120440

$ oc explain PodMonitor.spec.podMetricsEndpoints.tlsConfig
KIND:     PodMonitor
VERSION:  monitoring.coreos.com/v1

RESOURCE: tlsConfig <Object>

DESCRIPTION:
     TLS configuration to use when scraping the endpoint.

FIELDS:
   ca	<Object>
     Certificate authority used when verifying server certificates.

   cert	<Object>
     Client certificate to present when doing client-authentication.

   insecureSkipVerify	<boolean>
     Disable target certificate validation.

   keySecret	<Object>
     Secret containing the client key file for the targets.

   serverName	<string>
     Used to verify the hostname for the targets.
```

Reply:
Thanks! I'll create a jira for this. Since this affects core OpenShift (and is not unique to OSD or ROSA), I think this will be best addressed by one of the core OpenShift writers.